[2022-12-11 15:19:17,350] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task manual__2022-12-11T15:19:09.011585+00:00 [queued]>
[2022-12-11 15:19:17,379] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task manual__2022-12-11T15:19:09.011585+00:00 [queued]>
[2022-12-11 15:19:17,380] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 15:19:17,381] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 15:19:17,382] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 15:19:17,416] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): write_diff_task> on 2022-12-11 15:19:09.011585+00:00
[2022-12-11 15:19:17,428] {standard_task_runner.py:52} INFO - Started process 2442 to run task
[2022-12-11 15:19:17,444] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'write_diff_task', 'manual__2022-12-11T15:19:09.011585+00:00', '--job-id', '869', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpc8x8rsm3', '--error-file', '/tmp/tmp6f_nyo7g']
[2022-12-11 15:19:17,446] {standard_task_runner.py:80} INFO - Job 869: Subtask write_diff_task
[2022-12-11 15:19:17,541] {task_command.py:369} INFO - Running <TaskInstance: first_dag.write_diff_task manual__2022-12-11T15:19:09.011585+00:00 [running]> on host 7e39590a7807
[2022-12-11 15:19:17,718] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=write_diff_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T15:19:09.011585+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T15:19:09.011585+00:00
[2022-12-11 15:19:17,755] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 15:19:17,776] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=write_diff_task, execution_date=20221211T151909, start_date=20221211T151917, end_date=20221211T151917
[2022-12-11 15:19:17,822] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 15:19:17,869] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
